examples/imitation_learning_openbot/run_trained_policy.py

```diff
@@ -200,15 +165,15 @@
                 plt.imsave(os.path.realpath(os.path.join(image_dir, "%04d.jpg"%i)), obs_final_color)
 
                 # populate buffer and result data file
-                state_data[i] = obs["state_data"]
+                state_data[i] = np.concatenate((obs["location"], np.deg2rad(obs["rotation"])), axis=None)
                 df_result = pd.DataFrame({"left_ctrl"    : action[0],
```

Comment:
- `df_path_actual`
- convert to meters and radians
- fill in the data frame directly from `obs`, no need to construct an intermediate `state_data` variable

Reply:
Updated this